include/nuttx/virtio/virtio.h

```diff
@@ -0,0 +1,169 @@
+/****************************************************************************
+ * include/nuttx/virtio/virtio.h
```

Comment:
should we change file name to virito-mmio.h since it's tightly couple with mmio transport? Or i's better:

1. Split mmio specific macro and function to virtio-mmio.h
2. Keep the common virtio macro and function in virtio.h
3. Ensure virto net driver only use virtio macro and function not virtio mmio.

Reply:
OK. I will rename the file name to virtio-mmio.h.